cluster/src/test/java/org/apache/iotdb/cluster/query/reader/mult/RemoteMultSeriesReaderTest.java

```diff
@@ -182,100 +174,103 @@ public void testDefaultBatchStrategySelect() {
-                  @Override
-                  public void fetchMultSeries(
-                      RaftNode header,
-                      long readerId,
-                      List<String> paths,
-                      AsyncMethodCallback<Map<String, ByteBuffer>> resultHandler)
-                      throws TException {
-                    if (failedNodes.contains(node)) {
-                      throw new TException("Node down.");
-                    }
-
-                    new Thread(
-                            () -> {
-                              Map<String, ByteBuffer> stringByteBufferMap = Maps.newHashMap();
-                              if (batchUsed) {
-                                paths.forEach(
-                                    path -> {
-                                      stringByteBufferMap.put(path, ByteBuffer.allocate(0));
-                                    });
-                              } else {
-                                batchUsed = true;
-
-                                for (int i = 0; i < batchData.size(); i++) {
-                                  stringByteBufferMap.put(
-                                      paths.get(i), generateByteBuffer(batchData.get(i)));
-                                }
-
-                                resultHandler.onComplete(stringByteBufferMap);
-                              }
-                            })
-                        .start();
-                  }
-
-                  @Override
-                  public void queryMultSeries(
-                      MultSeriesQueryRequest request, AsyncMethodCallback<Long> resultHandler)
-                      throws TException {
-                    if (failedNodes.contains(node)) {
-                      throw new TException("Node down.");
-                    }
-
-                    new Thread(() -> resultHandler.onComplete(1L)).start();
-                  }
-                };
-              }
-            });
+    //    ClusterIoTDB.getInstance()
+    //        .setClientProvider(
+    //            new DataClientProvider(new Factory()) {
+    //              @Override
+    //              public AsyncDataClient getAsyncDataClient(Node node, int timeout) throws
+    // IOException {
+    //                return new AsyncDataClient(null, null, node, null) {
+    //                  @Override
+    //                  public void fetchMultSeries(
+    //                      RaftNode header,
+    //                      long readerId,
+    //                      List<String> paths,
+    //                      AsyncMethodCallback<Map<String, ByteBuffer>> resultHandler)
+    //                      throws TException {
+    //                    if (failedNodes.contains(node)) {
+    //                      throw new TException("Node down.");
+    //                    }
+    //
+    //                    new Thread(
+    //                            () -> {
```

Comment:
same

Reply:
Should be replaced by setClientManager().